.github/workflows/go.yml

```diff
@@ -24,8 +24,6 @@ on:
             - .github/workflows/lint-rust/action.yml
             - .github/workflows/install-valkey/action.yml
             - .github/json_matrices/build-matrix.json
-    workflow_dispatch:
```

Comment:
Please, restore - it allows to run CI on demand - without creating a PR or patching this file.

Reply:
Done